dev-haskell/reflex/reflex-0.8.0.0.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
```

Comment:
Same thing here, I suggest inserting `#hackport: flags: +split-these` to ease package maintenance when bumping versions.